--- tutos.orig/apache.conf	Tue Sep 17 12:30:40 2002
+++ tutos/apache.conf	Thu Oct 24 12:03:44 2002
@@ -66,9 +66,9 @@
 #
 # make tutos an aliased Directory
 #
-Alias /tutos "/usr/share/tutos"
+Alias /tutos "/home/services/httpd/html/tutos"
 
-<Directory "/usr/share/tutos">
+<Directory "/home/services/httpd/html/tutos">
     # Remove next line when already defined in httpd.conf
     # .pinc files must be interpreted by PHP to void password stealing !
     AddType application/x-httpd-php .pinc
--- tutos.orig/php/config_default.pinc	Tue Sep 17 12:30:40 2002
+++ tutos/php/config_default.pinc	Thu Oct 24 12:29:47 2002
@@ -35,7 +35,7 @@
 #
 # The Database user who runs TUTOS
 #
-$tutos[dbuser][0]   = "wwwrun";
+$tutos[dbuser][0]   = "tutos";
 #
 # The DB Users password
 #
@@ -99,7 +99,7 @@
 # ATTENTION: On Windows Change this to something your system understands !!!
 #            Like : $tutos[sessionpath] = "c:\\Temp";
 # 
-$tutos[sessionpath] = "/tmp";
+$tutos[sessionpath] = "/var/run/php";
 #
 # Logo for the headline
 # (Change this to your company logo or whatever you like)
@@ -108,16 +108,16 @@
 # NOTE: a value in the database entry will override these settings !!
 #        call adminpage -> select DB Name -> Modify -> change -> save
 # 
-#$tutos[logo] = "html/tutos_small.gif";
-$tutos[logo] = "http://sourceforge.net/sflogo.php?group_id=8047&amp;type=1";
-#$tutos[logolink] = "html/copyright.html";
-$tutos[logolink] = "http://www.tutos.org";
+$tutos[logo] = "html/tutos_small.gif";
+#$tutos[logo] = "http://sourceforge.net/sflogo.php?group_id=8047&amp;type=1";
+$tutos[logolink] = "html/copyright.html";
+#$tutos[logolink] = "http://www.tutos.org";
 #
 # The default timezone
 # This is the timezone where your database is running
 # For timezone names see near the end of this file.
 #
-$tutos[defaultTZ] = "Europe/Berlin";
+$tutos[defaultTZ] = "Europe/Warsaw";
 #
 # Proxy Definition
 #
@@ -255,7 +255,7 @@
 # Mode 1 = via sendmail
 # Mode 2 = via SMTP
 #
-$tutos[mailmode] = 2;
+$tutos[mailmode] = 1;
 # 
 # for auth use pop before smtp
 # (only useful in mailmode 2)
@@ -269,7 +269,7 @@
 # Where to find sendmail
 # Set this to something nonexistant to disable mailing
 #
-$tutos[sendmail]  = "/usr/lib/sendmail";
+$tutos[sendmail]  = "/usr/sbin/sendmail";
 #
 # Mode 2
 # The name or IP of the SMTP server
@@ -358,8 +358,8 @@
 # Here you can add whatever you want (default is the first one)
 #
 $tutos[currencies] = array();
+$tutos[currencies][] = "PLN";
 $tutos[currencies][] = "EUR";
-$tutos[currencies][] = "DM";
 $tutos[currencies][] = "US$";
 $tutos[currencies][] = "Lit";
 $tutos[currencies][] = "AUS$";
